ggshield/cmd/iac/scan/diff.py

```diff
@@ -91,26 +62,16 @@ def scan_diff_cmd(
         directory = Path().resolve()
     update_context(ctx, exit_zero, minimum_severity, ignore_policies, ignore_paths)
 
-    if pre_commit:
-        ref = "HEAD"
-        staged = True
-    elif pre_push:
-        ref = "@{upstream}"
-        staged = False
-    elif pre_receive:
-        ref = "HEAD"
-        staged = False
-
     result = iac_scan_diff(ctx, directory, ref, staged)
-    output_handler = create_output_handler(ctx)
-    if isinstance(result, IaCSkipScanResult):
-        return output_handler.process_skip_diff_scan()
-    scan = IaCDiffScanCollection(id=str(directory), result=result)
-    return output_handler.process_diff_scan(scan)
+    return display_iac_scan_diff_result(ctx, directory, result)
 
 
 def iac_scan_diff(
-    ctx: click.Context, directory: Path, ref: str, include_staged: bool
+    ctx: click.Context,
+    directory: Path,
+    ref: str,
+    include_staged: bool = False,
+    current: str = None,
```

Comment:
```suggestion
    current: Optional[str] = None,
```

Reply:
🏝 I might be wrong be I think, this is not added from the parent branch. Should I modify this here?

Reply:
I think you added it, you can change it here anyway